MCKillSwitch/MCKillSwitchAlert.m

```diff
@@ -150,14 +150,32 @@ - (void)hideAlertWithCompletion:(MCKillSwitchAlertBlock)completion {
 }
 
 + (UIViewController *)topMostViewController {
-    UIViewController *topMostViewController = UIApplication.sharedApplication.delegate.window.rootViewController;
+    UIViewController *topMostViewController = [self rootViewController];
     while (topMostViewController.presentedViewController != nil) {
         topMostViewController = topMostViewController.presentedViewController;
     }
     
     return topMostViewController;
 }
 
++ (UIViewController *)rootViewController {
+    UIViewController *rootViewController;
+    if (@available(iOS 13.0, *)) {
+        UIScene *scene = [[UIApplication.sharedApplication.connectedScenes allObjects] firstObject];
```

Comment:
Go all the way: `UIApplication.sharedApplication.connectedScenes.allObjects.firstObject`

Reply:
Dots, dots everywhere! :allthethings:

Dire que ça existait même pas il y a pas si longtemps...

Reply:
Y a 8 ans...

Reply:
Fixed :oldtimeyparrot: